### PR TITLE
Updating azblob to remove directory after removing backup

### PR DESCRIPTION
**Pull request by dkhenry**
The current azblob backup plugin doesn't delete the "folder" object when removing backup's. To do that you need to delete all the snapshots. If backup's have been created and deleted with previous versions of this plugin the folders may require manual deletion.

### CODE REVIEW

**Comment by deepthi** [comment]
linter is unhappy, that needs to be fixed.

File: Makefile [hunk 1]
```diff
@@ -37,6 +37,10 @@ ifdef VT_EXTRA_BUILD_FLAGS
 export EXTRA_BUILD_FLAGS := $(VT_EXTRA_BUILD_FLAGS)
 endif
 
+ifndef VTROOT
```
Comment (deepthi) [inline Makefile:40]: Why is this change needed?

Comment (dkhenry) [reply Makefile:40]: Its used in multiple places in the Makefile, but without it being defined those targets will never work. `make install` and `make install-testing` and `make proto` require it to be defined.

**Review by deepthi — APPROVED** [review]
LGTM

**Comment by deepthi** [comment]
@dkhenry can you create a PR to merge this to release-8.0?

File: Makefile [hunk 2]
```diff
@@ -69,7 +73,7 @@ endif
 install: build
 	# binaries
 	mkdir -p "$${PREFIX}/bin"
-	cp "$${VTROOT}/bin/"{mysqlctld,vtorc,vtctld,vtctlclient,vtgate,vttablet,vtworker,vtbackup} "$${PREFIX}/bin/"
+	cp "$${VTROOT}/bin/"{mysqlctl,mysqlctld,vtorc,vtctld,vtctlclient,vtgate,vttablet,vtworker,vtbackup} "$${PREFIX}/bin/"
```
Comment (enisoc) [inline Makefile:76]: Why did you add a new binary to the install target?

Comment (dkhenry) [reply Makefile:76]: that binary is used in the examples to start up mysql.
